cloudbuild.yaml

```diff
@@ -115,6 +115,17 @@ steps:
   dir: "build"
   args: ["ensure-build-sdk-image-base"]
 
+#
+# Preventing Broken PR Merges in CI
+#
+- name: "make-docker"
+  id: test-gen-crd-client
+  waitFor:
+    - pull-build-image
+    - pull-build-sdk-base-image  
```

Comment:
```suggestion
```

I don't think we need the sdk-base-image at this stage, so we can likely drop this.

Reply:
I've removed this line.